pkg/shared/accountmgmtutil/ams.go

```diff
@@ -87,7 +88,7 @@ func GetOrgQuotas(f *factory.Factory, spec *remote.AmsConfig) (*OrgQuotas, error
 		return nil, err
 	}
 
-	var standardQuotas, marketplaceQuotas, trialQuotas, evalQuotas []QuotaSpec
+	var standardQuotas, marketplaceQuotas, trialQuotas, evalQuotas, enterpriseQuotas []QuotaSpec
```

Comment:
this should be refactored later on and base of the logic by the billing model information that's returned by KFM for each supported instance type

Reply:
reflected in the code as a comment and noted 👍